cameo/flux_analysis/simulation.py

```diff
@@ -226,7 +228,7 @@ def create_objective(model, variables):
 
         cache.add_objective(create_objective, None, cache.variables.values())
 
-        solution = model.solve()
+        solution = model.optimize()
```

Comment:
Maybe, this is just a replacement for the now removed `model.solve`. Error handling there is scheduled to change so will be updated here when that happens as well.